build.gradle

```diff
@@ -21,6 +21,62 @@ allprojects {
     }
 }
 
+ext.versions = [
+        hadoop: '2.6.1',
+        hive  : '1.1.0',
+        restli: '1.15.7',
+        slf4j : '1.7.18',
+]
+
+ext.deps = [
+        // External dependencies
+        assertj             : 'org.assertj:assertj-core:3.8.0',
+        collections         : 'commons-collections:commons-collections:3.2.2',
+        commonsLang         : 'commons-lang:commons-lang:2.6',
+        dbcp2               : 'org.apache.commons:commons-dbcp2:2.1.1',
+        dbutils             : 'commons-dbutils:commons-dbutils:1.5',
+        fileupload          : 'commons-fileupload:commons-fileupload:1.2.1',
+        guava               : 'com.google.guava:guava:21.0',
+        guice               : 'com.google.inject:guice:4.1.0',
+        h2                  : 'com.h2database:h2:1.4.193',
+        hadoopAnnotations   : "org.apache.hadoop:hadoop-annotations:" + versions.hadoop,
```

Comment:
Never mind, no strong opinion by me and can't really argue why I would prefer the $ way. I guess I just personally find it more readable when you can more easily see where a string starts and where it ends. Maybe someone even shared that view, because it was done with $ originally :)